app/routes/verkiezingen.js

```diff
@@ -7,13 +7,18 @@ import { BESTUURSEENHEID_CLASSIFICATIECODE_GEMEENTE } from 'frontend-lmb/utils/w
 export default class VerkiezingenRoute extends Route {
   @service currentSession;
   @service session;
+  @service router;
 
   beforeModel(transition) {
     this.session.requireAuthentication(transition, 'login');
 
     if (!this.currentSession.canAccessMandaat) {
       this.router.transitionTo('index');
     }
+
+    if (!this.currentSession.showLegislatuurModule) {
+      this.router.transitionTo('index');
+    }
```

Comment:
Nitpick: would do it like this, the outcome of the if statement is the same, but I do like the readability of your version (if it stays for two check ;p)  

```suggestion
    if (
      !this.currentSession.canAccessMandaat ||
      !this.currentSession.showLegislatuurModule
    ) {
      this.router.transitionTo('index');
    }
```